Header header
float64[] coef_opt
float64[] domain
float64 residual_avg
float64 residual_std
string type
